--- ntop/Makefile.am	2004-01-16 12:28:28.000000000 -0600
+++ ntop/Makefile.am	2004-02-17 14:24:07.000000000 -0600
@@ -201,7 +201,7 @@
 	@echo "       If we hang here, then you are probably "
 	@echo "       running make not GNU make (gmake)"
 	@echo ""
-	@groff -mandoc -Tascii $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -~]//g' > ntop.txt
+	@groff -mandoc -Tascii $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -\~]//g' > ntop.txt
 	@echo "$@ done !"
 	@echo "-----------------------------------------------"
 	@echo ""
@@ -220,7 +220,7 @@
 	@echo "       If we hang here, then you are probably "
 	@echo "       running make not GNU make (gmake)"
 	@echo ""
-	@groff -mandoc -Thtml $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -~]//g' > html/ntop.html
+	@groff -mandoc -Thtml $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -\~]//g' > html/ntop.html
 	@cp -f html/ntop.html ntop.html
 	@echo "$@ done !"
 	@echo "-----------------------------------------------"
@@ -239,7 +239,7 @@
 	@echo "       If we hang here, then you are probably "
 	@echo "       running make not GNU make (gmake)"
 	@echo ""
-	@@AWK@ -f utils/faqtohtml.awk $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -~]//g' > html/faq.html
+	@@AWK@ -f utils/faqtohtml.awk $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -\~]//g' > html/faq.html
 	@cp -f html/faq.html faq.html
 	@echo "$@ done !"
 	@echo "-----------------------------------------------"
--- ntop/Makefile.in	2004-01-16 12:28:28.000000000 -0600
+++ ntop/Makefile.in	2004-02-17 14:24:18.000000000 -0600
@@ -1115,7 +1115,7 @@
 	@echo "       If we hang here, then you are probably "
 	@echo "       running make not GNU make (gmake)"
 	@echo ""
-	@groff -mandoc -Tascii $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -~]//g' > ntop.txt
+	@groff -mandoc -Tascii $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -\~]//g' > ntop.txt
 	@echo "$@ done !"
 	@echo "-----------------------------------------------"
 	@echo ""
@@ -1133,7 +1133,7 @@
 	@echo "       If we hang here, then you are probably "
 	@echo "       running make not GNU make (gmake)"
 	@echo ""
-	@groff -mandoc -Thtml $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -~]//g' > html/ntop.html
+	@groff -mandoc -Thtml $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -\~]//g' > html/ntop.html
 	@cp -f html/ntop.html ntop.html
 	@echo "$@ done !"
 	@echo "-----------------------------------------------"
@@ -1152,7 +1152,7 @@
 	@echo "       If we hang here, then you are probably "
 	@echo "       running make not GNU make (gmake)"
 	@echo ""
-	@@AWK@ -f utils/faqtohtml.awk $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -~]//g' > html/faq.html
+	@@AWK@ -f utils/faqtohtml.awk $< | LANG=C sed 's/_^H//g' | LANG=C sed 's/^H[ -\~]//g' > html/faq.html
 	@cp -f html/faq.html faq.html
 	@echo "$@ done !"
 	@echo "-----------------------------------------------"
