--- ntop/configure.in.orig	Tue Jul  9 18:49:48 2002
+++ ntop/configure.in	Tue Jul  9 18:50:04 2002
@@ -61,7 +61,7 @@
 HAS_ERROR=
 HAS_WARNING=
 
-NTOP_VERSION_EXTRA=""
+NTOP_VERSION_EXTRA="SourceForge RPM"
 
 echo
 echo "Welcome to $PACKAGE_NAME, Version $PACKAGE_VERSION $NTOP_VERSION_EXTRA"
--- ntop/configure.orig	Mon Apr 14 17:54:20 2003
+++ ntop/configure	Mon Apr 14 17:54:35 2003
@@ -1352,7 +1352,7 @@
 HAS_ERROR=
 HAS_WARNING=
 
-NTOP_VERSION_EXTRA=""
+NTOP_VERSION_EXTRA="SourceForge RPM"
 
 echo
 echo "Welcome to $PACKAGE_NAME, Version $PACKAGE_VERSION $NTOP_VERSION_EXTRA"
